src/MongoDB.Driver.Legacy/MongoCollection.cs

```diff
@@ -1702,11 +1702,12 @@ private MapReduceResult MapReduce(IClientSessionHandle session, MapReduceArgs ar
         /// <typeparam name="TDocument">The type of the document.</typeparam>
         /// <param name="args">The args.</param>
         /// <returns>Multiple enumerators, one for each cursor.</returns>
+        [Obsolete("ParallelScanAs was deprecated in version 4.1.")]
         public ReadOnlyCollection<IEnumerator<TDocument>> ParallelScanAs<TDocument>(ParallelScanArgs<TDocument> args)
         {
             return UsingImplicitSession(session => ParallelScanAs(session, args));
         }
-
+        [Obsolete("ParallelScanAs was deprecated in version 4.1.")]
```

Comment:
Oops, looks like I forgot to mention in the style guide that we like to have newlines between method declarations :)